apps/dashboard/src/locales/en.json

```diff
@@ -298,6 +298,7 @@
     "Email address": "Email address",
     "First name": "First name",
     "Last name": "Last name",
+    "nickname": "Nickname",
```

Comment:
`Nickname`

Reply:
changed in https://github.com/GEWIS/sudosos-frontend/pull/287/commits/395268f8a4f67c96119b5698dc24bc12a2471960